src/app/app.component.ts

```diff
@@ -62,6 +68,11 @@ export class AppComponent {
       .subscribe(res => this.hideToolbar = true);
 
     userService.registerRefresh();
+
+    // listen for appointments
+    this.getAppointmentStatus();
+    this.appointmentSubscription = Observable.interval(60000)
```

Comment:
Why this? Can't we use websockets?

Reply:
We'd need some kind of timer/schedule on the server that also finds the right socket and then notifies it.